setup.py

```diff
@@ -81,6 +81,10 @@
     'hf_transfer==0.1.3',
 ]
 
+extra_deps['databricks'] = [
+    'mosaicml[databricks]',
```

Comment:
didn't specify a version because I assume it default to whatever composer version we're using.

Reply:
yeah, using the composer version is ok. we only pull it out if we need to be more restrictive than composer for some reason